packages/frontend/public/index.html

```diff
@@ -12,6 +12,15 @@
       href="https://fonts.googleapis.com/css2?family=Azeret+Mono:wght@200..700&display=swap"
     />
     <title>trusted setup</title>
+    <meta property="og:title" content="Unirep Trusted Setup" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://ceremony.unirep.io" />
+    <meta property="og:image" content="og.png" />
+    <meta property="og:site_name" content="Unirep Ceremony" />
+    <meta
+      property="og:description"
+      content="Generate your verse to the UniRep Ceremony to knitting together the constellation that unveil our journey."
```

Comment:
replace `content` with:

'Generate your own verse and help secure the UniRep protocol.'